src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.vb

```diff
@@ -50,6 +81,32 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                 AssemblyVersionMajorTextBox, AssemblyVersionMinorTextBox, AssemblyVersionBuildTextBox, AssemblyVersionRevisionTextBox}
         End Sub
 
+        ''' <summary>
+        ''' This checks the properties for licenses to determine the initial state of the licensing section.
+        ''' Currently, If both are set, it will default to just enabling the PackageLicenseExpression.
+        ''' In the future, it might be a good idea to have a warning and only output one value, but the nuget error after packing is helpful.
+        ''' TryGetNonCommonPropertyValue will get the property if it is set even if it is empty, but because the empty properties 
+        ''' are ignored elsewhere, they are ignored here.
+        ''' </summary>
+        Private Sub InitializeLicensing()
+            Dim PackageLicenseFileSet = TryCast(TryGetNonCommonPropertyValue(GetPropertyDescriptor("PackageLicenseFile")), String)
+            If (PackageLicenseFileSet IsNot Nothing And PackageLicenseFileSet IsNot "") Then
```

Comment:
Consider using `AndAlso`. Not a big deal though.

Reply:
That is a good idea, I will change it